WORKSPACE

```diff
@@ -35,6 +35,11 @@ workspace(name = "com_github_stratum_stratum")
 #       Please do not push changes to this section upstream.
 # ---------------------------------------------------------------------------
 
+#local_repository(
+#    name = "com_github_p4lang_PI_np4",
+#    path = "/home/craigsid/np4_install/PI",
+#)
+
```

Comment:
remove?

Reply:
yeah maybe, I guess I left it as an example of how to include dev versions of the PI code.  Perhaps I could change my userid to say <userid> ? and add a comment to this effect?

Reply:
removed